src/js/utils/background.js

```diff
@@ -85,20 +85,21 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
       styles.push(css`
         background-color: ${backgroundColor};
         ${(!background.opacity || background.opacity !== 'weak') &&
-          `color: ${
+          `color: ${normalizeColor(
             textColor[
               background.dark || colorIsDark(backgroundColor) ? 'dark' : 'light'
-            ]
-          };`}
+            ] || textColor,
+            theme,
+          )};`}
       `);
     }
     if (background.dark === false) {
       styles.push(css`
-        color: ${textColor.light};
+        color: ${textColor.light || textColor};
       `);
     } else if (background.dark) {
       styles.push(css`
-        color: ${textColor.dark};
+        color: ${textColor.dark || textColor};
```

Comment:
Is this one missing `normalizeColor` as well?

Reply:
Good catch! fixed